resources/building-mobile-extensions/logging-from-extension.md

```diff
@@ -1,19 +1,19 @@
 # Extension logging
 
-We recommend that you use the Experience Platform SDK logging API to print a message from the extension code. The Experience Platform SDKs provide the `setLogLevel` API, which is used to set one of the following logging modes:
+You should use the Experience Platform SDK logging API to print a message from the extension code. The Experience Platform SDKs provide the `setLogLevel` API, which is used to set one of the following logging modes:
```

Comment:
This is not a requirement, but a recommendation, let's keep `It is recommended that you use..